src/modules/submissions.js

```diff
@@ -483,7 +483,25 @@
 		return this.text;
 	};
 
-	AFCH.Text.prototype.cleanUp = function ( isAccept ) {
+	function getDraftArticleSubcatsInner( cat ) {
+		return new mw.Api().get( { action: 'query', list: 'categorymembers', cmtitle: 'Category:' + cat, cmtype: 'subcat' } ).then( function ( data ) {
+			return $.when.apply( $, data.query.categorymembers.map( function ( e ) {
+				var txt = e.title.replace( /Category:(.*?\s*)/gi, '$1' );
+				return getDraftArticleSubcatsInner( txt ); // changed it to return apiCall here
```

Comment:
nitpick: delete the comment on this line ("changed it...") and two lines down ("starting with cat here..."); below in getDraftArticleSubcats there also seem to be left-over comments that should also be removed